### PR TITLE
docs: Update the UI design in the README file #176

### LINKED ISSUE #176: Update the UI design in the `README` file
**Current State:**
Screenshot from 2021-09-14 18-40-04

**Desired State:**
Screenshot from 2021-09-14 18-38-52

### PR DESCRIPTION
**Pull request by Aayushi-Mittal**
Fixes #176

## Proposed Changes

  - Updated the UI in README file.

## Screenshots
image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/185"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +43,7 @@ Your url will be `http://linkfree.eddiehub.org/<yourusername>`, for example http
 
 Your `avatar` URL should take the format of `https://github.com/<yourusername>.png`.
 
+
```
Comment (schmelto) [inline README.md:46]:
```suggestion
```

**Review by FayasNoushad — APPROVED** [review]

**Review by aayush89890 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
-2. clone project
-3. navigate to project directory `cd LinkFree`
-4. install dependencies with `npm install`
-5. run `npm start`
+1. Fork project
+2. Clone project
+3. Navigate to project directory `cd LinkFree`
+4. Install dependencies with `npm install`
+5. Run `npm start`
+
```
Comment (schmelto) [inline README.md:16]:
```suggestion
```

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks for your contribution 🚀 and thanks all for reviewing

**Comment by Aayushi-Mittal** [comment]
> Thanks for your contribution 🚀 and thanks all for reviewing

Thank you! :)

**Pull request merged**